src/components/duelling-picklist/duelling-picklist.component.tsx

```diff
@@ -88,38 +108,4 @@ const DuellingPicklist = ({
   );
 };
 
-DuellingPicklist.propTypes = {
-  ...marginPropTypes,
-  /**
-   * Content of the component, should contain two Picklist children
-   * and a PicklistDivider
-   */
-  children: (props, propName) => {
-    const prop = props[propName];
-    let error;
-
-    if (
-      !prop ||
-      !Array.isArray(prop) ||
-      prop.filter((el) => el.type === Picklist).length !== 2
-    ) {
-      error = new Error(
-        `\`${propName}\` must have two \`${Picklist.displayName}\`s`
-      );
-    }
-
-    return error;
-  },
-  /** Indicate if component is disabled */
-  disabled: PropTypes.bool,
-  /** Place for components like Search or Filter placed above the left list */
-  leftControls: PropTypes.node,
-  /** Place for components like Search or Filter placed above the right list */
-  rightControls: PropTypes.node,
-  /** Left list label */
-  leftLabel: PropTypes.string,
-  /** Right list label */
-  rightLabel: PropTypes.string,
-};
-
 export default DuellingPicklist;
```

Comment:
```suggestion
export default DuellingPicklist;
DuellingPicklist.displayName = "DuellingPicklist";
```

Reply:
There's no need for the displayName assignment here as the DuellingPicklist is not wrapped in forwardRef